### ISSUE TITLE
NPM Install Fails On 2.8.1

### ISSUE THREAD

**Issue by willashe** [froala/react-froala-wysiwyg#86]
I'm working on a project that has `react-froala-wysiwyg@^2.8.1` as a dependency.  For some reason, running `npm install` for the project stopped working yesterday, having worked just fine the previous day.  The failure appears to be due to 2.8.1 having Babel 6.23.0 as a dependency.  Version 2.8.0 does not have this dependency and installs correctly.  Even just running `npm install react-froala-wysiwyg@^2.8.1` with no other dependencies throws the same warning/error:

The first warning I get from NPM has to do with react-froala-wysiwyg 2.8.1 depending on the deprecated version of Babel:
```
npm WARN deprecated babel@6.23.0: In 6.x, the babel package has been deprecated in favor of babel-cli. Check https://opencollective.com/babel to support the Babel maintainers
```

Then NPM fails and quits with the following output:
```
npm ERR! path /var/www/node_modules/babel/lib/cli.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/var/www/node_modules/babel/lib/cli.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```

This seems clearly to be an issue with 2.8.1.  Is anyone else able to install this version?  Am I missing something?  Also not sure how this was working on Monday, since the Babel dependency was [introduced on April 22](https://github.com/froala/react-froala-wysiwyg/commit/4d99ec0e4c5b626b0a75dea9a388da1ab9d5f101) (possibly something to do with NPM cache?).  Thanks in advance for any insight.

**Comment by willashe** [comment]
Ok, bit more insight into what’s going on.  I’m running all of my npm operations from inside a docker container.  I must have pulled the container down and spun it back up, effectively clearing the npm cache which contained a cached version of the deprecated Babel version (installed/cached prior to the repo being taken down).  This is why the install now fails since it’s trying to pull a fresh copy of a package from a repo that no longer exists.

TL;DR: I believe the deprecated Babel dependency should be removed.

**Comment by kristian-puccio** [comment]
I've just run into this issue as well.
Needs to be fixed asap I think
